phpunit-integration-multisite.xml.dist

```diff
@@ -1,11 +1,14 @@
 <?xml version="1.0" ?>
 <phpunit
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
   bootstrap="tests/phpunit/integration/bootstrap.php"
   backupGlobals="false"
   colors="true"
+  failOnRisky="true"
   convertErrorsToExceptions="true"
   convertNoticesToExceptions="true"
   convertWarningsToExceptions="true"
```

Comment:
Worth doing [this](https://twitter.com/jrf_nl/status/1441823275550527494) at the same time.

Reply:
This change here actually addresses the issue described in that tweet. See the linked core ticket, https://core.trac.wordpress.org/ticket/54183.

This change here is option "fail on risky tests caused by deprecation warnings"

So we're all set there

Reply:
Good.